### PR TITLE
added dp begginer problemset

### PR DESCRIPTION
**Pull request by Farrius**
I found a  problemset with classical dp problems. I think it is a suitable contest for begginers. You decide if it is starred or not.

### CODE REVIEW

**Comment by dongliu0426** [comment]
Maybe be consistent with the file naming [here](https://github.com/Farrius/cp/tree/master/codeforces/gym_contest/dp_div3_practice).

**Review by dongliu0426 — COMMENTED** [review]
Idk about labeling the problems "Beginner" because some require some dp techniques.

File: content/4_Gold/Intro_DP.mdx [hunk 1]
```diff
@@ -183,7 +183,13 @@ The next few modules provide examples of some classical problems, or Dynamic Pro
...
 		starred
 	>
-		Some tasks are beyond the scope of Gold.
+	</Resource>
+	<Resource
+		source="CF@Codeforces"
+		title="Begginer DP Contest"
+		url="https://codeforces.com/blog/entry/86532"
+	>
+		Begginer friendly classical problems. Some tasks requires input/output files. <a href="https://github.com/Farrius/cp/tree/master/codeforces/gym_contest/dp_div3_practice">Here</a> you can check the solutions.
```
Comment (dolphingarlic) [inline content/4_Gold/Intro_DP.mdx:192]: "Begginer" should be spelt as "beginner"

File: content/4_Gold/Intro_DP.mdx [hunk 2]
```diff
@@ -183,7 +183,13 @@ The next few modules provide examples of some classical problems, or Dynamic Pro
 		url="https://atcoder.jp/contests/dp/tasks"
 		starred
 	>
-		Some tasks are beyond the scope of Gold.
```
Comment (dolphingarlic) [inline content/4_Gold/Intro_DP.mdx:186]: Also, why do you delete this line?

**Review by dolphingarlic — CHANGES_REQUESTED** [review]

File: content/4_Gold/Intro_DP.mdx (hunk 2 above)
Comment (Farrius) [reply content/4_Gold/Intro_DP.mdx:186]: Ups, not intended

File: content/4_Gold/Intro_DP.mdx (hunk 1 above)
Comment (Farrius) [reply content/4_Gold/Intro_DP.mdx]: > Maybe be consistent with the file naming [here](https://github.com/Farrius/cp/tree/master/codeforces/gym_contest/dp_div3_practice).

Sure, now the name of the files are a, b, c, d...  in order to be more clear.

File: content/4_Gold/Intro_DP.mdx [hunk 3]
```diff
@@ -183,7 +183,14 @@ The next few modules provide examples of some classical problems, or Dynamic Pro
...
+	</Resource>
+	 	Some tasks are beyond the scope of Gold.
```
Comment (dolphingarlic) [inline content/4_Gold/Intro_DP.mdx:187]: I think you switched the order of these two lines

File: content/4_Gold/Intro_DP.mdx [hunk 4]
```diff
@@ -183,7 +183,14 @@ The next few modules provide examples of some classical problems, or Dynamic Pro
...
+	<Resource
+		source="CF@Codeforces"
+		title="Beginner DP Contest"
+		url="https://codeforces.com/blog/entry/86532"
+	>
+		Beginner friendly classical problems. Some tasks requires input/output files and some dp techniques. <a href="https://github.com/Farrius/cp/tree/master/codeforces/gym_contest/dp_div3_practice">Here</a> you can check the solutions.
```
Comment (dolphingarlic) [inline content/4_Gold/Intro_DP.mdx:193]:
```suggestion
		Beginner-friendly classical problems. Some tasks requires input/output files. <a href="https://github.com/Farrius/cp/tree/master/codeforces/gym_contest/dp_div3_practice">Here</a> you can check the solutions.
```

**Review by dolphingarlic — CHANGES_REQUESTED** [review]

File: content/4_Gold/Intro_DP.mdx (hunk 3 above)
Comment (Farrius) [reply content/4_Gold/Intro_DP.mdx]: Now is fixed, in my host looks good.

**Pull request merged**